docker-compose.yml

```diff
@@ -0,0 +1,51 @@
+version: '2.4'
+
+networks:
+  # haven is a traefik network that is setup elsewhere (and beyond the scope of this example)
+  haven:
+    external: true
+    name: haven
```

Comment:
can we do this example without using `treafik`? I'm getting the following error when trying to run this locally without it:

```
network haven declared as external, but could not be found
```

Reply:
One can
1. add traefik config into this compose file 
2. Or just comment that stuff out and bind the port(s)
3. Or actually setup traefik

On a host machine with multiple services traefik is a great tool for configurable reverse proxy routing.